webapp/components/create_comment.jsx

```diff
@@ -150,6 +151,8 @@ class CreateComment extends React.Component {
         Client.createPost(
             post,
             (data) => {
+                AsyncClient.getPosts(this.props.channelId);
```

Comment:
This is going to hurt performance, especially on mobile. Not sure why we need to get all the posts again?